blog/on_retros/README.md

```diff
@@ -0,0 +1,272 @@
+
+3) Should be a venue for experimentation.
+
+Because you are already working in increments of time (I prefer 1-week sprints since that is the smallest increment my teams have been able to deliver value in while minimizing the latency on our "feedback loops" but do whatever your team is comfortable with) it's relatively low risk trying new things. Worst case scenario is you lose one sprint. So brainstorm ideas, try them, and review them next retro. Then iterate again!
+
+4) Should result in small, well-defined action items.
+
+Like engineering sprint work, the smaller and more well-defined the action item the more likely it will actually get done. Action items that basically amount to "try not to have as much rollover" probably won't be nearly as effective as "let's break down all 5+ point tickets into smaller tickets before bringing them into the sprint" or "let's configure jira to allow a maximum number of 4 in-progress tickets at a time."
+
+5) The primary goal should be to build the team that the member's want to work on. Iteratively turn your job into your [dream job!](https://www.youtube.com/watch?v=gAMu5YMOZn0&ab_channel=DivaDan)
+
+Every engineering team I have ever worked on has been full of smart capable people who love to write code and want to get work done, but they have problems getting in their way. Too many meetings, too much rollover, confusion on what the user story is asking for, feeling isolated from team members, etc. If it's bothering you or getting in your way, bring it to retro and try to get it fixed! If you think there is a "better way" there is no reason we shouldn't shoot for it.
+
+### Here are common problems I see in running retros
+
+More detailed patterns will be included later.
+
+#### Lack of engagement
+I've heard people say retros are "a waste of time", "nothing ever improves," "no one talks," "it's boring," "no one pays attention," etc.
+
+Agile's biggest flaw, like with OOP and many other Good Things, is that it is easy to do it badly, think it's terrible, and throw it away.
+
+If the team doesn't have faith in the retro process, they won't engage with it, and it becomes a self-fulfilling prophecy. And there can be a variety of reasons for this
+1) Internalization that work is "unpleasant" and always will be
+2) Learned helplessness
+3) Feeling like one's experiences or ideas are not valuable enough to be shared.
+
+These are hard problems to solve, but here are some of my recommendations
+1) Evangelize the process. Talk up how the point of the retro is to make everyone happier at work.
+2) Scope down the action items and track them to make sure they actually get done. Show the team that real change comes from retros.
```

Comment:
Who is responsible for doing this?

Reply:
added another short blurb about it being "the team" and the team should be "self organizing and engineer led"